pkg/ebpf/ingress_node_firewall_loader.go

```diff
@@ -158,23 +199,50 @@ func ingessNodeFwAttach(objs bpfObjects, ifacesName []string, pinDir string, isD
 				log.Fatalf("could not attach XDP program: %s", err)
 				return err
 			}
-
-			lpinDir := path.Join(pinDir, ifaceName+"-link")
-			if err := l.Pin(lpinDir); err != nil {
-				log.Fatalf("failed to pin link to pinDir %s: %s", lpinDir, err)
+			lPinDir := path.Join(infc.pinPath, ifaceName+"_link")
+			if err := l.Pin(lPinDir); err != nil {
+				log.Fatalf("failed to pin link to pinDir %s: %s", lPinDir, err)
```

Comment:
Why not just return error? Fatal will kill the program.

Reply:
yeah converted all to return err this was just part of PoC